cartridges/int_klaviyo/cartridge/scripts/klaviyo/viewedProductHelpers.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+/* API Includes */
+var PromotionMgr = require('dw/campaign/PromotionMgr');
+var Promotion = require('dw/campaign/Promotion');
+var StringUtils = require('dw/util/StringUtils');
+var Logger = require('dw/system/Logger');
+
+/* Script Modules */
+var app = require('*/cartridge/scripts/app');
+var klaviyoUtils = require('*/cartridge/scripts/klaviyo/utils');
+
+
+function getProductPrices(product) {
```

Comment:
Would you mind adding a docstring to this helper method w/ a short explanation of why its needed

Reply:
Done.